### ISSUE TITLE
referrerpolicy on <script> elements

### ISSUE THREAD

**Issue by gregbaker** [validator/validator#817]
The validator doesn't accept the `referrerpolicy` attribute on `<script>` elements, but my reading of [the spec of the script element](https://html.spec.whatwg.org/multipage/scripting.html#the-script-element) says it should be allowed (except for data block script elements).

This page fails validation but passes when the `referrerpolicy` is removed.
```html
<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="UTF-8" />
<title>Page Title</title>
<script src="code.js" referrerpolicy="origin"></script>
</head>
<body>
</body>
</html>
```

**Comment by sideshowbarker** [comment]
Yep, I see it’s been in the HTML spec now for almost a year —

https://github.com/whatwg/html/commit/0d28f8fc3a7c272f6d148f647ead29965a56ab60

Dunno how I didn’t get it added to the checker at that time — so thanks very much for catching it and taking time to report it. I’ll get it added to the checker today or tomorrow.

**Comment by sideshowbarker** [comment]
Thanks for reporting this — fix pushed to https://validator.w3.org/nu/
